packages/frontend/src/components/manage/HouseSettings.tsx

```diff
@@ -76,11 +68,12 @@ const HouseSettings: React.FC<HouseSettingsProps> = (props) => {
               });
             }}
             color="secondary"
-            width="60%"
+            // className='w-full'
```

Comment:
Not sure if it's important but did you want to leave in this commented out code `className='w-full'`? Same for the other instances of it.

Reply:
Nice catch! Thanks for letting me know, I was going to delete them but there were too many files haha